server/client_server.go

```diff
@@ -63,66 +55,64 @@ func (client jiraServerClient) GetIssueInfo(projectID string) (*ProjectIssueInfo
 	return &issues, response, err
 }
 
-func (client jiraServerClient) GetProjectInfo(currentVersion semver.Version, pivotVersion semver.Version, options *jira.GetQueryOptions) (*jira.CreateMetaInfo, *jira.Response, error) {
-	var info *jira.CreateMetaInfo
-	var resp *jira.Response
-	var req *http.Request
+func (client jiraServerClient) GetProjectInfoWithNewEndpoints(options *jira.GetQueryOptions) (*jira.CreateMetaInfo, *jira.Response, error) {
```

Comment:
I think this sound better
```suggestion
func (client jiraServerClient) GetProjectInfoWithForPivotJiraVersion(options *jira.GetQueryOptions) (*jira.CreateMetaInfo, *jira.Response, error) {
```

Reply:
I personally feel that `GetProjectInfoWithForPivotJiraVersion` doesn't seems good.

Reply:
GetProjectInfoForPivotJiraVersion then is better because the new endpoint doesn't make sense suppose in the future they change this then which will be a new endpoint

Reply:
Yeah, `GetProjectInfoForPivotJiraVersion` seems good